data/ships.txt

```diff
@@ -2136,6 +2176,60 @@ ship "Osprey"
 
 
 
+ship "Prospector"
+	sprite "ship/mining carrier"
+	attributes
+		category "Heavy Freighter"
+		"cost" 2340000
```

Comment:
Exact same hull price as the Hauler II. And judging by some of the other stats, that's what you based it off of. :)
I'd lower this closer to 2 mil, maybe a little lower.